fastf1/core.py

```diff
@@ -2694,8 +2747,10 @@ def split_qualifying_sessions(self) -> List[Optional["Laps"]]:
                     session_suspended = False
             elif row['Status'] == 'Aborted':
                 session_suspended = True
+            elif row['Status'] == 'Finished':
```

Comment:
This handles the case when a qualifying session isn't restarted after a red flag